src/Marten/Util/TypeMappings.cs

```diff
@@ -310,17 +310,6 @@ public static bool HasTypeMapping(Type memberType)
             return ResolvePgType(memberType) != null || memberType.IsEnum;
         }
 
-        public static string ApplyCastToLocator(this string locator, EnumStorage enumStyle, Type memberType)
```

Comment:
Can we just mark this method as obsolete rather than removing it to make it a non-breaking change in terms of binary compatibility? Also, include a comment that it will be removed in v4.0.

Reply:
done